libraries/StorageManager/StorageManager.cpp

```diff
@@ -95,7 +96,8 @@ const StorageManager::StorageArea StorageManager::layout_copter[STORAGE_NUM_AREA
     { StorageParam,    8192,  1280},
     { StorageRally,    9472,   300},
     { StorageFence,    9772,   256},
-    { StorageMission, 10028,  6228}, // leave 128 byte gap for expansion
+    { StorageMission,  10028,  5204}, // leave 128 byte gap for expansion
+    { StorageCANDNA,   15232,  1024},
```

Comment:
i'll do a separate PR to enable the 2nd 16 region of FRAM on boards that have 32k RAMTRON (most boards) if they are F7 or H7 boards.